cmd/system-probe/modules/tcp_queue_tracer.go

```diff
@@ -42,5 +46,7 @@ func (t *tcpQueueLengthModule) Register(httpMux *module.Router) error {
 }
 
 func (t *tcpQueueLengthModule) GetStats() map[string]interface{} {
-	return nil
+	return map[string]interface{}{
+		"last_check": atomic.LoadInt64(&t.lastCheck),
```

Comment:
can we convert the timestamp back to a `time.Time`?

Reply:
maybe via a `GetTime( *int64) time.Time` helper on the `module` package

Reply:
Wouldn't a human-readable format would make more sense in a flare status?

Reply:
> Wouldn't a human-readable format would make more sense in a flare status?

That is happening, but in the template rendering via the `formatUnixTime` helper. See https://github.com/DataDog/datadog-agent/pull/9896/files#diff-a206f97de83d10e3baedae31258228e0fa41dd882c2a81972dc4d069e9ee6a20R23 for example.

Reply:
Oh, I see. Thanks